EKF/terrain_estimator.cpp

```diff
@@ -99,7 +99,7 @@ void Ekf::fuseHagl()
 		_hagl_innov = pred_hagl - meas_hagl;
 
 		// calculate the observation variance adding the variance of the vehicles own height uncertainty and factoring in the effect of tilt on measurement error
-		float obs_variance = fmaxf(P[9][9], 0.0f) + sq(_params.range_noise / _R_rng_to_earth_2_2);
+		float obs_variance = fmaxf(P[9][9], 0.0f) + (sq(_params.range_noise) + sq(_params.range_noise_scaler * _range_sample_delayed.rng)) * _R_rng_to_earth_2_2;
```

Comment:
The  height error when tilted will also be affected by deviation of the terrain away from the flat earth assumption and errors in the tilt angle estimate which will increase the error with bank angle. This means that it is not practical to scale height error with tilt. I will remove the R_rng_to_earth_2_2 scaler.